controllers/duration.js

```diff
@@ -1,10 +1,28 @@
+<<<<<<< HEAD
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
const logDuration = require("../models/duration");
```